src/analytics/formChangeTracker/formChangeTracker.ts

```diff
@@ -26,23 +31,33 @@ export class FormChangeTracker extends FormTracker {
       return false;
     }
 
-    const submitUrl = this.getSubmitUrl(form);
+    let element: HTMLLinkElement = event.target as HTMLLinkElement;
+
+    // Ensure element is an <a> tag with "Change" text content and is not a lang toggle link
+    if (
+      element.tagName !== "A" ||
+      !this.isChangeLink(element) ||
+      element.hasAttribute("hreflang")
```

Comment:
For this check, we probably only need the `element.hasAttribute("hreflang")`

Reply:
But this check is not only for excluding lang toggle links but links which aren't change links and elements which aren't links